src/ws/hooks/item.test.ts

```diff
@@ -28,15 +29,17 @@ describe('Ws Item Hooks', () => {
   });
 
   describe('useItemUpdates', () => {
-    const item = ITEMS[0];
+    const item = ITEMS.toArray()[0];
     const itemId = item.id;
     const itemKey = buildItemKey(itemId);
     const channel = { name: itemId, topic: TOPICS.ITEM };
-    const newItem = { ...item, description: 'new description' };
+    var changedItem = item.toJS();
+    changedItem = { ...changedItem, description: 'new description' };
+    const newItem: RecordOf<Item> = convertJs(changedItem);
```

Comment:
I think you can just update the value with `update` so you don't need to cast it twice.